staging/src/k8s.io/component-base/logs/logs.go

```diff
@@ -27,9 +27,9 @@ import (
 	"k8s.io/klog/v2"
 )
 
-const logFlushFreqFlagName = "log-flush-frequency"
+const LogFlushFreqFlagName = "log-flush-frequency"
```

Comment:
Why `logFlushFreqFlagName` to `LogFlushFreqFlagName`? Some other pkg import it?

Reply:
[staging/src/k8s.io/component-base/config/validation/validation.go:89](https://github.com/kubernetes/kubernetes/pull/99693/files#diff-5d0601b20c5ccb8c7abf87e7e0dd477eeeb6862cb186c96595c156aced23a5c8R89) uses it